converters/toZigbee.js

```diff
@@ -4241,25 +4241,28 @@ const converters = {
     heiman_ir_remote: {
         key: ['send_key', 'create', 'learn', 'delete', 'get_list'],
         convertSet: async (entity, key, value, meta) => {
+            const options = {
+                manufacturerCode: null,
```

Comment:
I gues this forces the `manufacturerCode` to null instead of https://github.com/Koenkk/zigbee-herdsman/blob/a9819cb485fad01c21d75cdc05879fd8f5854f5e/src/zcl/definition/cluster.ts#L4154 ?